README.md

```diff
@@ -9,32 +9,36 @@ currently supports these checks:
 
 - `build-settings-externalized`
 
-    This mode ensures there are no settings configured at the project level,
-    instead encouraging the use of xcconfig files.
+This mode ensures there are no settings configured at the project level,
```

Comment:
Curious what happened to all the indents here.  Had you run swiftformat?

Reply:
I ran control + i, but yeah better to roll back.